website/blog/2024-07-25-ci-migration.mdx

```diff
@@ -0,0 +1,78 @@
+---
+slug: ci-migration
+title: "Optimizing and Migrating Velox CI Workloads to Github Actions"
+authors: [jwujciak, kgpai]
+tags: [tech-blog, packaging]
+---
+
+<figure>
+ <img src="/img/abstract-white-background.jpg" height= "100%" width="100%"/>
+</figure>
+
+## TL;DR
+
+In late 2023, the Meta OSS (Open Source Software) Team requested all Meta teams to move the CI deployments from CircleCI to Github Actions. [Voltron Data](http://voltrondata.com) and Meta in collaboration migrated all the deployed Velox CI jobs. For the year 2024, Velox CI spend was on track to overshoot the allocated resources by a considerable amount of money. As part of this migration effort, the CI workloads were consolidated and optimized by Q2 2024, bringing down the projected 2024 CI spend by 51%.
+
+## Velox’s Continuous Integration Workload
+
+Continuous Integration (CI) is crucial for Velox’s success as an open source project as it helps protect from bugs and errors, reduces likelihood of conflicts and leads to increased community trust in the project. This is to ensure the Velox builds works well on a myriad of system architectures, operating systems and compilers - along with from the ones used internally at Meta. The OSS build version of Velox also supports additional features that aren't used internally in Meta (for example, support for Cloud blob stores, etc.).
+
+When a pull request is submitted to Velox, the following jobs are executed:
+
+1. Linting and Formatting workflows:
+    a. Header checks
+    b. License checks
+    c. Basic Linters 
```

Comment:
Ordered lists can only be numeric. I think an unordered list (-/*) is sufficient from a contents perspective as well.
![image](https://github.com/user-attachments/assets/6ecc19c0-fb47-4018-8685-6df336a5b869)

Reply:
thanks, updated